tools/matRad_gammaIndex.m

```diff
@@ -119,7 +122,12 @@
    
 % interpolate if necessary
 if n > 0
-    cubex2 = interp3(cubex2,n,'cubic');
+    switch env
+        case 'MATLAB'
+            cubex2 = interp3(cubex2,n,'cubic');
+        case 'OCTAVE'
+            warning('3-D interpolation not yet supported by octave. Function will continue without interpolation.')
```

Comment:
@becker89: 3-D interpolation with interp3 is supported in Octave, only the method 'cubic' that is not (see [https://www.gnu.org/software/octave/doc/v4.2.0/Multi_002ddimensional-Interpolation.html#XREFinterp3](https://www.gnu.org/software/octave/doc/v4.2.0/Multi_002ddimensional-Interpolation.html#XREFinterp3)).